lib/rails/generators/alchemy/base.rb

```diff
@@ -32,7 +32,7 @@ def template_engine
       end
 
       def load_alchemy_yaml(name)
-        YAML.safe_load(ERB.new(File.read("#{Rails.root}/config/alchemy/#{name}")).result, [Regexp], [], true)
+        YAML.safe_load(ERB.new(File.read("#{Rails.root}/config/alchemy/#{name}")).result, [Regexp, Symbol], [], true)
```

Comment:
Good catch and thanks!

Since this method is used in all our generators we should allow all the classes that element and page layouts needs; that are:

`Date`, `Regexp` and `Symbol`